.changeset/selfish-games-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+'highlight.run': minor
```

Comment:
```suggestion
'highlight.run': patch
```
since this is not affecting the end user behavior (since we are dropping the `visited-url` events on our backend but still recording new fields from the navigate events)